src/storage/v2/config.hpp

```diff
@@ -49,7 +49,7 @@ struct Config {
     uint64_t wal_file_flush_every_n_tx{100000};
 
     bool snapshot_on_exit{false};
-
+    bool restore_replicas_on_startup{false};
```

Comment:
should this default to true? in memgraph.cpp it is set to true. I don't know what our conventions are but this feels like an opportunity for misunderstandings if it sort of has two defaults, but the one in memgraph.cpp is "actually default" but bound later.